### DISCUSSION TITLE
Is it possible to adjust the width of the "Setting up the course generator" window?

### DISCUSSION THREAD

**Discussion by BanShieRU** [Courseplay/Courseplay_FS22#2725] [Q&A] [answered]
The problem is the following. I'm playing on three monitors and Nvidia Surround has the monitor frame compensation set. When I open the course generator window, it stretches to the center and left monitor and all the settings values are hidden behind the monitor frame. Is there a way to change the width or position of this window? This window is the only one in the game that is displayed in this way. Thank you in advance. Sorry for the English, I'm using a translator.

**Comment by schwiti6190** [comment, marked as answer]
@BanShieRU Could you give it another try with this version #2726 ?

**Reply by BanShieRU** [reply]
Thank you for your help. I have changed the specified files. When loading any of the saves, the download progress freezes at 85%.  CoursePlay version 7.3.0.2

**Reply by Tensuko** [reply]
Try this:
[FS22_Courseplay.zip](https://github.com/Courseplay/Courseplay_FS22/files/12446357/FS22_Courseplay.zip)

**Reply by BanShieRU** [reply]
Thank you very much! Apparently I moved something to the file incorrectly. Now everything is working as it should.

**Reply by Tensuko** [reply]
Fix will be part of 7.3.0.3
